pkgs/by-name/te/teamspeak3/package.nix

```diff
@@ -0,0 +1,143 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  fetchzip,
+  makeWrapper,
+  makeDesktopItem,
+  glib,
+  qt5,
+  libsForQt5,
+  xkeyboard_config,
+  perl,
+  libcxx,
+  autoPatchelfHook,
+}:
+
+let
+  arch = "amd64";
```

Comment:
I don't think this is useful; we parameterise source from it, but there's no other build right now, and there's a handful of other places we reference the arch (e.g. when going into the directory once we've unpacked the installer).

Reply:
I just retained this from the original drv as I didn't have a strong opinion on this and saw it as fine as is.